test/views/templates/payments/WhatYouOweChargeTemplateSpec.scala

```diff
@@ -182,4 +182,114 @@ class WhatYouOweChargeTemplateSpec extends ViewBaseSpec {
       }
     }
   }
+
+  "Rendering a default surcharge charge" when {
+
+    def generateModel(overdue: Boolean): OpenPaymentsModel = {
+      OpenPaymentsModel(
+        vatDefaultSurcharge,
+        BigDecimal(300.00),
+        LocalDate.parse("2000-05-10"),
+        LocalDate.parse("2000-02-01"),
+        LocalDate.parse("2000-03-28"),
+        "#004",
+        overdue
+      )
+    }
+
+    "the payment is not overdue" should {
+
+      val model = generateModel(overdue = false)
+      lazy val view = views.html.templates.payments.whatYouOweCharge(model, 0, Some(true))
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      "display the correct description text" in {
+        elementText(Selectors.description) shouldBe
+          "this is a surcharge for late payment of your 1 February to 28 March 2000 return"
+      }
+
+      "display the correct pay text" in {
+        elementText(Selectors.payText) shouldBe "Pay now"
+      }
+
+      "have the correct pay link context" in {
+        elementText(Selectors.payContext) shouldBe
+          "£300 , this is a surcharge for late payment of your 1 February to 28 March 2000 return"
+      }
+
+      "not display the view return link" in {
+        intercept[org.scalatest.exceptions.TestFailedException](element(Selectors.viewReturnLink))
+      }
+    }
+
+    "the payment is overdue" should {
+
+      val model = generateModel(overdue = true)
+      lazy val view = views.html.templates.payments.whatYouOweCharge(model, 0, Some(true))
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      println(document.body())
```

Comment:
Println statement here. Has this been left in accidentally?

Also, could you please squash commits just to avoid putting meaningless commits in the git history.